docs/user-guide/hypershift-agent-service-config.md

```diff
@@ -0,0 +1,141 @@
+# Hypershift zero worker deployment
+
+## Prerequisites
+1. Deploy hub cluster using [dev-scripts]((https://github.com/javipolo/openshift-assisted-installer-tests/tree/4515edfc9d60a587cabd3bcfdd77826cab8cc02d/ai-in-zero-worker-hypershift#deploy-an-l0-hub-cluster-using-dev-scripts))
+2. Hive [installation](https://github.com/javipolo/openshift-assisted-installer-tests/tree/4515edfc9d60a587cabd3bcfdd77826cab8cc02d/ai-in-zero-worker-hypershift#install-hypershift) 
+3. Create [hypershift cluster](https://github.com/javipolo/openshift-assisted-installer-tests/tree/4515edfc9d60a587cabd3bcfdd77826cab8cc02d/ai-in-zero-worker-hypershift#create-hypershift-cluster)
+
+## Deploying the hub cluster
+### Set up our environment
```

Comment:
probably worth referring to HyperShift official docs for additional info: https://hypershift-docs.netlify.app/how-to/agent/create-agent-cluster/